packages/app-desktop/gui/NoteListControls/NoteListControls.tsx

```diff
@@ -69,6 +71,73 @@ const SortOrderButtonsContainer = styled.div`
 
 function NoteListControls(props: Props) {
 	const searchBarRef = useRef(null);
+	const newNoteRef = useRef(null);
+	const newTodoRef = useRef(null);
+	const noteControlsRef = useRef(null);
+	const searchAndSortRef = useRef(null);
+
+	const [noteBtnText, setNoteBtnText] = useState('New note');
+	const [todoBtnText, setTodoBtnText] = useState('New to-do');
+	const [breakpoint, setBreakpoint] = useState('l');
+	const [iconNote, setIconNote] = useState('fas fa-plus');
+	const [iconTodo, setIconTodo] = useState('fas fa-plus');
+
+	useEffect(() => {
+		const breakpoints = [{ s: 135 }, { md: 189 }, { l: 470 }, { xl: 500 }];
```

Comment:
We have a `ValueMap`in packages/app-mobile/components/getResponsiveValue.ts. Could you maybe re-use this interface? You might have to refactor and move it under /lib first and then important from both app-mobile and here

Reply:
Actually "ValueMap" is such a poor name - could you see if can find a proper name for it, maybe whatever is used in responsive CSS?

Reply:
Technically this `ValueMap`  interface has the right shape, but it is used in a different way. It apparently contains values to be returned based on a window's width and breakpoints hardcoded in `getResponsiveValue.ts`.

Whereas what I'm doing here is simply finding the closest breakpoint to a width.

I could reuse it and rename it `Breakpoints` or something similar but the code in `getResponsiveValue.ts` would become very confusing with two sorts of breakpoints.

Or I could create an interface `Breakpoints`.

Reply:
Ok then please create a new interface specifically for this